src/System.Reflection.Emit/tests/AssemblyBuilder/AssemblyBuilderDefineDynamicModule1.cs

```
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Threading;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class AssemblyGetModuleTests
+    {
+        [Fact]
+        public void PosTest1()
+        {
+            AssemblyName myAsmName = new AssemblyName("TestAssembly1");
+            AssemblyBuilder myAssembly = AssemblyBuilder.DefineDynamicAssembly(myAsmName, AssemblyBuilderAccess.Run);
+            ModuleBuilder mb = TestLibrary.Utilities.GetModuleBuilder(myAssembly, "Module1");
+        }
+
+        [Fact]
+        public void PosTest2()
+        {
+            try
+            {
+                AssemblyName myAsmName = new AssemblyName("TestAssembly2");
+                AssemblyBuilder myAssembly = AssemblyBuilder.DefineDynamicAssembly(myAsmName, AssemblyBuilderAccess.Run);
+                ModuleBuilder myModuleBuilder = TestLibrary.Utilities.GetModuleBuilder(myAssembly, "Module1");
+                TypeBuilder myTypeBuilder = myModuleBuilder.DefineType("HelloWorld", TypeAttributes.Public);
+                ConstructorBuilder myConstructor = myTypeBuilder.DefineConstructor(
+                         MethodAttributes.Public, CallingConventions.Standard, new Type[] { });
+                ILGenerator myILGenerator = myConstructor.GetILGenerator();
+                myILGenerator.Emit(OpCodes.Ldarg_1);
+            }
+            catch (InvalidOperationException)
```

I think here should be a comment with explanation  why we eat this exception.